src/verifai/samplers/domain_sampler.py

```diff
@@ -114,8 +141,13 @@ def __init__(self, domain):
         super().__init__(domain)
 
     def nextSample(self, feedback=None):
-        sample = self.nextVector(feedback)
-        return self.domain.unstandardize(sample)
+        sample, info = self.nextVector(feedback)
+        # print(f'inside BoxSampler.nextSample and sample = {sample}')
+        return self.domain.unstandardize(sample), info
+    
+    def getSample(self):
+        sample, info = self.nextVector()
+        return self.domain.unstandardize(sample), info
 
     def nextVector(self, feedback=None):
```

Comment:
Need to replace with `getVector` method and have `nextVector` be defined in terms of it, like `getSample` and `nextSample` in `DomainSampler`. Then new `BoxSampler` classes need only define `getVector` (and possibly `update` -- it probably makes sense to have an `updateVector` [or something] method too since `update` will receive the full sample, rather than the standardized vector which comes out of `getVector`, and this would be inconsistent).